styles.css

```diff
@@ -0,0 +1,13 @@
+.buttons {
+  height:200px;
```

Comment:
for clean code and syntax sake... it should read 
height: 200px; 
width: 200px;
cursor: pointer;
border: none;
float: left;

there needs to be a space after the colon, between the key value pairs.